src/app/datapoints-graph/alarms-filtering/alarms-filter.component.ts

```diff
@@ -0,0 +1,220 @@
+import {
```

Comment:
we do, but not in this version of ngx-components. notice it is separated from other stuff, seems like Enio already predicted that it will be removed from here and we will replace this and alarm-event-selector with the one from ngx-components.

Reply:
@DawidSAG @jdreSAG  yes, as Jakub mentions, similar to the alarm-event-selector this will be removed once it is available in ngx-components, but for this version it is not..